src/common/client-app.ts

```diff
@@ -21,20 +21,28 @@ export var defaultConfiguration: AppConfiguration = {
 
 export function start(boardVM: IBoardVM, config?: AppConfiguration) {
   var app = new App(boardVM, config);
-  app.start();
+  config = utils.extend({ }, defaultConfiguration, config);

```

Comment:
This line ensures to set default values and overridden with custom ones, it should be executed before `var app = new App(boardVM, config);`, right?